styles.css

```diff
@@ -5,439 +5,1094 @@
-  font-family: 'Inter', sans-serif;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #979493;
-}
+  .exp2 {
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
 
-textarea {
-  width: 327px;
-  height: 114px;
-  padding: 0 10px;
-  border-style: none;
-  background-color: #fbf8f7;
-  color: #3c3a39;
-  font-family: 'Inter', sans-serif;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-}
+  .function:hover {
+    background: #1c1a19;
+    color: #fff;
+  }
 
-.btn-center {
-  margin: 1px 0 0;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  margin-bottom: 10px;
-}
+  .contact-form {
+    background-color: #fff;
+    margin: 0;
+  }
 
-.title2 {
-  font-family: 'inter', sans-serif;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  text-align: center;
-  color: #172b4d;
-  padding: 8rem 0 0;
-}
+  .container3 {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding-bottom: 120px;
+  }
 
-.footer {
-  height: 80px;
-  background: #fff;
-  border-top: 1px solid #dfe1e6;
-  padding-top: 20px;
-}
+  form {
+    padding: 2rem 0 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
 
-.footer-links {
-  padding-top: 1rem;
-  display: flex;
-  justify-content: center;
-  gap: 14px;
-}
+  form li + li {
+    margin-top: 1em;
+  }
+
+  .first-last {
+    display: none;
+  }
+
+  input {
+    padding: 0 10px;
+    width: 327px;
+    height: 48px;
+    border-style: none;
+    border-bottom: 2px solid #dbd8d7;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #979493;
+  }
+
+  .textarea1 textarea {
+    width: 327px;
+    height: 114px;
+    padding: 0 10px;
+    border-style: none;
+    background-color: #fbf8f7;
+    color: #3c3a39;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .textarea2 {
+    display: none;
+  }
+
+  .btn-center {
+    margin: 1px 0 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+  }
+
+  .title2 {
+    font-family: 'inter', sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+    padding: 8rem 0 0;
+  }
+
+  .footer {
+    height: 80px;
+    background: #fff;
+    border-top: 1px solid #dfe1e6;
+    padding-top: 20px;
+  }
+
+  .footer-links {
+    padding-top: 1rem;
+    display: flex;
+    justify-content: center;
+    gap: 14px;
+  }
 
-.footer hr {
-  width: 134px;
-  height: 5px;
-  background-color: #0d151e;
-  margin: 3px auto;
-  border-radius: 1px;
+  .footer hr {
+    width: 134px;
+    height: 5px;
+    background-color: #0d151e;
+    margin: 3px auto;
+    border-radius: 10px;
+  }
+
+  .footer-hr {
+    padding-top: 10px;
+  }
 }
 
```

Comment:
### MOBILE VERSION


- Hey, please, the mobile version is having a horizontal scroll effect which is not a great experience for the user. please, let's fix this to improve the user experience of your portfolio website.